solr/core/src/java/org/apache/solr/update/SolrIndexConfig.java

```diff
@@ -381,7 +381,16 @@ private MergeScheduler buildMergeScheduler(IndexSchema schema) {
         if (maxThreadCount == null) {
           maxThreadCount = ((ConcurrentMergeScheduler) scheduler).getMaxThreadCount();
         }
-        ((ConcurrentMergeScheduler)scheduler).setMaxMergesAndThreads(maxMergeCount, maxThreadCount);
+        ConcurrentMergeScheduler cmScheduler = (ConcurrentMergeScheduler)scheduler;
```

Comment:
Space before `scheduler`

Reply:
For their code style, yeah good point (personally I dislike that style, but not my choice!)

Reply:
I thought it might be code style, but 2 lines above it adds a space 🤷‍♂️ 

Actually now I've read that line again, I think you can move the variable up a bit and re-use it there too?

Reply:
oh .. good idea, there's 2 other casts